src/neuroconv/datainterfaces/ophys/femtonics/femtonicsdatainterface.py

```diff
@@ -0,0 +1,352 @@
+        sampling_freq = femtonics_metadata["sampling_frequency_hz"]
+        imaging_plane = metadata["Ophys"]["ImagingPlane"][0]
+        imaging_plane["imaging_rate"] = sampling_freq
+
+        # Add geometric transformations to ImagingPlane description
+        geometric_transformations = femtonics_metadata.get("geometric_transformations", {})
+        if geometric_transformations:
+            imaging_plane = metadata["Ophys"]["ImagingPlane"][0]
+            desc = imaging_plane.get("description", "")
+            gt_parts = []
+            if geometric_transformations.get("translation") is not None:
+                gt_parts.append(f"translation: {geometric_transformations['translation']}")
+            if geometric_transformations.get("rotation") is not None:
+                gt_parts.append(f"rotation: {geometric_transformations['rotation']}")
+            if geometric_transformations.get("labeling_origin") is not None:
+                gt_parts.append(f"labeling_origin: {geometric_transformations['labeling_origin']}")
+            if gt_parts:
+                desc = (desc + " " if desc else "") + "Geometric transformations: " + ", ".join(gt_parts)
+                imaging_plane["description"] = desc.strip()
+
+        return metadata
+
+    @staticmethod
+    def get_available_channels(
+        file_path: FolderPathType, session_name: str = None, munit_name: str = None
+    ) -> list[str]:
+        """
+        Get available channels in the specified session/unit combination.
+
+        Parameters
+        ----------
+        file_path : str or Path
+            Path to the .mesc file.
+        session_name : str, optional
+            Name of the MSession to use (e.g., "MSession_0").
+            If None, uses the first available session.
+        munit_name : str, optional
+            Name of the MUnit within the session (e.g., "MUnit_0").
+            If None, uses the first available unit.
+
+        Returns
+        -------
+        list of str
+            List of available channel names.
+        """
+        Extractor = FemtonicsImagingInterface.get_extractor()
+
+        # Handle session_name to session_index conversion
+        if session_name is None:
+            session_keys = Extractor.get_available_sessions(file_path=file_path)
+            if not session_keys:
+                raise ValueError("No sessions found")
+            session_index = 0  # Use first session if none specified
+        else:
+            session_keys = Extractor.get_available_sessions(file_path=file_path)
+            if session_name not in session_keys:
+                raise ValueError(f"Session '{session_name}' not found. Available sessions: {session_keys}")
+            session_index = session_keys.index(session_name)
+
+        # Handle munit_name to munit_index conversion
+        if munit_name is None:
+            unit_keys = Extractor.get_available_units(file_path=file_path, session_index=session_index)
```

Comment:
```suggestion
            unit_keys = cls.get_available_munits(file_path=file_path, session_index=session_index)
```

We can use the class static method here, right? So we don't need to reference the extractor and to use names instead of indices.

Reply:
yes, done.